libiocage/lib/Jail.py

```diff
@@ -661,22 +711,21 @@ def _teardown_mounts(self):
                 )
 
     def _resolve_name(self, text):
+
+        if (text is None) or (len(text)==0):
```

Comment:
I know, you know I hate parenthesis, but I think this is a good example. It makes de-compacting that hard on my eyes. Instead of `if test is None or len(text) == 0`, it's easier to see those elements to me. I won't mark as needs changes, because it's your personal style ;)

But just felt I had to say it atleast! :D